### PR TITLE
Unittest on check_partials in subsystems on flops_based side 

### PR DESCRIPTION
**Pull request by xjjiang**
### Summary

- Add necessary unit tests on partials within `subsystems` on the `flops_based` side.
- Add missing unit tests on output values within `subsystems` on the `flops_based` side. Some components (especially in `mass`) are too simple and hence I decided not to create unit tests.
- The supersonic case in `subsystems/aerodynamics/flops_based/test/test_design.py` needs further investigation. The partial derivative of `mission:design:mach` with respect to `aircraft:wing:thickness_to_chord` does not seem to be correct. 
- The `ComputedDragTest` in `subsystems/aerodynamics/flops_based/test/test_drag.py` needs further investigation. The value of `Dynamic.Mission.DRAG` seem to be too small.
- Delete `TotalDragCoeff` class from `aerodynamics/flops_based/drag.py` because it is never used nor tested. 
- Removed a bunch of unused variables from:
-- `subsystems/mass/flops_based/avionics.py]`
-- `subsystems/mass/flops_based/test/test_avionics.py`
-- `subsystems/mass/flops_based/unusable_fuel.py`
-- `subsystems/mass/flops_based/test/test_unusable_fuel.py`
- Added the default value of `Mission.Summary.FUEL_FLOW_SCALER` into `models/large_single_aisle_2/large_single_aisle_2_altwt_FLOPS_data.py` to avoid an unnecessary warning.
- Added `Settings.VERBOSITY=0` option to suppress some warning messages.

### Related Issues

None

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

**Review by jkirk5 — APPROVED** [review]

File: aviary/subsystems/aerodynamics/flops_based/test/test_mux_component.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def test_mux(self):
         prob.set_val(Aircraft.Nacelle.LAMINAR_FLOW_LOWER, np.array([0.10]))
 
         prob.run_model()
-        prob.check_partials(compact_print=True, method="cs")
+        # prob.check_partials(compact_print=True, method="cs")
```
Comment (johnjasa) [inline aviary/subsystems/aerodynamics/flops_based/test/test_mux_component.py:61]: Is this purposefully commented out?

File: aviary/subsystems/aerodynamics/flops_based/test/test_mux_component.py [hunk 2]
```diff
@@ -133,7 +133,7 @@ def test_mux_multiengine(self):
         prob.set_val(Aircraft.Nacelle.LAMINAR_FLOW_LOWER, np.array([0.10, 0.04]))
 
         prob.run_model()
-        prob.check_partials(compact_print=True, method="cs")
+        # prob.check_partials(compact_print=True, method="cs")
```
Comment (johnjasa) [inline aviary/subsystems/aerodynamics/flops_based/test/test_mux_component.py:136]: Is this purposefully commented out?

File: aviary/subsystems/aerodynamics/flops_based/test/test_skinfriction_coef.py [hunk 3]
```diff
@@ -16,7 +16,7 @@ def test_derivs(self):
         nc = 3
 
         machs = np.array([.2, .3, .4, .5, .6, .7, .75, .775, .8, .825, .85, .875])
-        alts = np.linspace(41000, 41000, n)
+        # alts = np.linspace(41000, 41000, n)
```
Comment (johnjasa) [inline aviary/subsystems/aerodynamics/flops_based/test/test_skinfriction_coef.py:19]: remove this if not needed?

**Review by johnjasa — APPROVED** [review]
Thanks for these changes, Xun! Some quick questions but I'm approving so I don't hold up the PR.

File: aviary/subsystems/aerodynamics/flops_based/test/test_design.py [hunk 4]
```diff
@@ -11,38 +11,98 @@
...
+        derivs = prob.check_partials(out_stream=None, method="cs")
         assert_check_partials(derivs, atol=1e-12, rtol=1e-12)
 
-    def test_derivs_subsonic(self):
+        assert_near_equal(
+            prob.get_val(Mission.Design.MACH), [0.671145], 1e-6)
+        assert_near_equal(
+            prob.get_val(Mission.Design.LIFT_COEFFICIENT), [0.683002], 1e-6)
+
+    def test_derivs_subsonic2(self):
```
[the remark this answers is not in the document]
Comment (xjjiang) [reply aviary/subsystems/aerodynamics/flops_based/test/test_design.py:88]: Done. Now, the four tests are:
`test_derivs_supersonic1`
`test_derivs_supersonic2`
`test_derivs_subsonic1`
`test_derivs_subsonic1`

File: aviary/subsystems/aerodynamics/flops_based/test/test_design.py [hunk 5]
```diff
@@ -53,10 +113,48 @@ def test_derivs_subsonic(self):
...
 
         derivs = prob.check_partials(out_stream=None, method="cs")
+        assert_check_partials(derivs, atol=1e-12, rtol=1e-12)
+
+        assert_near_equal(
+            prob.get_val(Mission.Design.MACH), [0.671145], 1e-6)
+        assert_near_equal(
+            prob.get_val(Mission.Design.LIFT_COEFFICIENT), [0.683002], 1e-6)
+
+    def test_derivs_subsonic3(self):
```
[the remark this answers is not in the document]
Comment (xjjiang) [reply aviary/subsystems/aerodynamics/flops_based/test/test_design.py:123]: Done.

File: aviary/subsystems/aerodynamics/flops_based/test/test_design.py [hunk 6]
```diff
@@ -11,38 +11,98 @@
 class DesignMCLTest(unittest.TestCase):
 
     def test_derivs_supersonic(self):
```
[the remark this answers is not in the document]
Comment (xjjiang) [reply aviary/subsystems/aerodynamics/flops_based/test/test_design.py:13]: Done

File: aviary/subsystems/aerodynamics/flops_based/test/test_mux_component.py (hunk 1 above)
Comment (xjjiang) [reply aviary/subsystems/aerodynamics/flops_based/test/test_mux_component.py:61]: I should have kept it. It is uncommented out now.

File: aviary/subsystems/aerodynamics/flops_based/test/test_mux_component.py (hunk 2 above)
Comment (xjjiang) [reply aviary/subsystems/aerodynamics/flops_based/test/test_mux_component.py:136]: I didn't understand why we do `check_partials` twice. Now, I keep it in.

File: aviary/subsystems/aerodynamics/flops_based/test/test_skinfriction_coef.py (hunk 3 above)
Comment (xjjiang) [reply aviary/subsystems/aerodynamics/flops_based/test/test_skinfriction_coef.py:19]: I removed all three lines on `alts`.

**Comment by xjjiang** [comment]
The supersonic case in `subsystems/aerodynamics/flops_based/test/test_design.py` is complete. In the `OpenMDAO` interpolation, the derivatives aren't always continuous if you interpolate right on one of the table points. By changing the value of Thickness to Chord from 0.04 to 0.04000001, the problem is solved.

**Comment by xjjiang** [comment]
The investigation on the `ComputedDragTest` in `subsystems/aerodynamics/flops_based/test/test_drag.py` is done. Parameters are tuned to get reasonable result.

**Pull request merged**